src/main/deltachat.js

```diff
@@ -444,11 +443,17 @@ class DeltaChatController {
 }
 
 function snakeCaseKeys (obj) {
-  const result = {}
-  Object.keys(obj).forEach(key => {
-    result[snakeCase(key)] = obj[key]
-  })
-  return result
+  return {
+    addr: obj.addr,
+    mail_user: obj.mailUser,
+    mail_pw: obj.mailPw,
+    mail_server: obj.mailServer,
+    mail_port: obj.mailPort,
+    send_user: obj.sendUser,
+    send_pw: obj.sendPw,
+    send_server: obj.sendServer,
+    send_port: obj.sendPort
```

Comment:
So this code is incomplete since I left out `mailSecurity` and `sendSecurity`. There are no corresponding properties with that name in core.

Reply:
yeah the corresponding key would be server_flags (we are translating that already with translate... something method)

Reply:
we should introduce a serverFlag variable and also do the snakecase stuff for it.